boards/components/src/sched/priority.rs

```diff
@@ -0,0 +1,36 @@
+//! Component for a priority scheduler.
+//!
+//! This provides one Component, PriorityComponent.
+//!
+//! Usage
+//! -----
+//! (Must be final lines in main.rs)
+//! ```rust
+//! let scheduler =
+//!     components::priority::PriorityComponent::new(board_kernel).finalize(());
+//! scheduler.kernel_loop(&imix, chip, Some(&imix.ipc), &main_cap);
```

Comment:
```suggestion
//! Note that `kernel_loop` never returns, thus this must end main.rs
//! ```rust
//! let scheduler =
//!     components::priority::PriorityComponent::new(board_kernel).finalize(());
//! scheduler.kernel_loop(&imix, chip, Some(&imix.ipc), &main_cap)
```